src/olympia/lib/tests/test_git.py

```diff
@@ -186,6 +187,60 @@ def test_extract_and_commit_from_version_set_git_hash():
     assert len(addon.current_version.git_hash) == 40
 
 
+@pytest.mark.django_db
+def test_delete(settings):
+    addon = addon_factory(file_kw={'filename': 'webextension_no_id.xpi'})
+    addon.current_version.update(git_hash='some hash')
```

Comment:
you can do all this in one statement with `addon = addon_factory(version_kw={'git_hash': 'some hash'}, file_kw={'filename': 'webextension_no_id.xpi'})`